helm_lib/README.md

```diff
@@ -11,3 +12,15 @@ For example:
 ```bash
 make version=0.0.2 update-lib-helm
 ```
+
+## Fix lib-helm bugs
+
+ATTENTION!
+
+Do not backport new minor and major releases of `lib-helm` to an existing (already released on the Alpha channel) Deckhouse release. 
+These releases may contain breaking changes. Also, you can't use the [backporting mechanism](https://github.com/deckhouse/deckhouse/wiki/Guidelines-for-working-with-PRs#backporting-a-pr)! 
+If you need to make a fix, follow the instructions below:
+- Release a patch release of `lib-helm` [as follows instruction](https://github.com/deckhouse/lib-helm/blob/main/README.md#fix-bug-in-minor-release-xx).
```

Comment:
```suggestion
- Release a patch release of `lib-helm` [as follows instruction](https://github.com/deckhouse/lib-helm/blob/main/README.md#fix-bug-in-minor-release-xx).
```
@name212 Incorrect link?

Reply:
Yep, we need to merge this [pr](https://github.com/deckhouse/lib-helm/pull/9)